Mathlib/Tactic/Linter/Papercut.lean

```diff
@@ -0,0 +1,126 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import Lean.Elab.Command
+import Lean.Linter.Util
+import Batteries.Data.List.Basic
+
+/-!
+# The papercut linter
+
+The papercut linter tries to ease new users into Lean.
+It flags certain design decisions that may be surprising or confusing.
+
+For instance,
+* `3 - 5 = 0` (`Nat`-subtraction is truncated subtraction),
+* `3 / 5 = 0` (`Nat`-division is rounded division),
+* `(3 : ℤ) / 5 = 0` (`Int`-division is rounded division),
+* `x / 0 = 0` for most types (division by `0` equals `0`).
```

Comment:
Sounds more like a garbage value linter than a papercut linter to me. There are other papercuts than just garbage values eg the fact that `Module` means all three of "semimodule", "module" and "vector space".